sites/friday/src/get-started/express/_index.md

```diff
@@ -1,4 +1,490 @@
+   ```
+
+Et voilà, your Express application is live!
+
+## Make changes to your project
+
+Now that your project is deployed, you can start making changes to it.
+For example, you might want to fix a bug or add a new feature.
+
+In your project, the `main` branch always represents the production environment.
+Other branches are for developing new features, fixing bugs, or updating the infrastructure.
+
+To make changes to your project, follow these steps:
+
+1. Create a new environment (a Git branch) to make changes without impacting production:
+
+   ```bash
+   {{% vendor/cli %}} branch feat-a
+   ```
+
+   This command creates a new local `feat-a` Git branch based on the main Git branch
+   and activates a related environment on {{< vendor/name >}}.
+   The new environment inherits the data (service data and assets) of its parent environment (the production environment here).
+
+2. Make changes to your project.
+
+   For example, edit the `./index.js` file and make the following changes:
+
+   ```html {location="index.js"}
+   app.get('/', async function(req, res){
+     // Make the output.
+     const outputString = `Hello, World! - A simple Express web framework template for {{% vendor/name %}}`
+     res.send(outputString);
+   });
+   ```
+
+3. Commit your changes:
+
+   ```bash
+   $ git add index.js
+   $ git commit -m "Update Hello world"
+   ```
+
+4. Deploy your changes to the `feat-a` environment:
+
+   ```bash
+   $ {{% vendor/cli %}} push
+   ```
+
+   Note that each environment has its own domain name.
+   To open the url of your new environment, run the following command:
+
+   ```bash
+   $ {{% vendor/cli %}} environment:url --primary
+   ```
+
+5. Iterate by changing the code, committing, and deploying.
+   When satisfied with your changes, merge them to the main branch, deploy,
+   and remove the feature branch:
+
+   ```bash
+   $ {{% vendor/cli %}} merge
+     Are you sure you want to merge feat-a into its parent, main? [Y/n] y
+   $ {{% vendor/cli %}} checkout main
+   $ git pull {{% vendor/cli %}} main
+   $ {{% vendor/cli %}} environment:delete feat-a
+   $ git fetch --prune
+   ```
+
+   {{< note >}}
+   Deploying to production was fast because the image built for the `feat-a` environment was reused.
+   {{< /note >}}
+
+   For a long running branch, to keep the code up-to-date with the main branch, use `git merge main` or `git rebase main`.
+   You can also keep the data in sync with the production environment by using `{{% vendor/cli %}} env:sync`.
+
+## Use a third-party Git provider
+
+When you choose to use a third-party Git hosting service, the {{< vendor/name >}} Git
+repository becomes a read-only mirror of the third-party repository. All your
+changes take place in the third-party repository.
+
+Add an integration to your existing third-party repository:
+
+- [BitBucket](/integrations/source/bitbucket.md)
+- [GitHub](/integrations/source/github.md)
+- [GitLab](/integrations/source/gitlab.md)
+
+## Next steps
```

Comment:
For standardization, it would be good to have the same buttons as in Paul and Thomas' PRs for Flask and Laravel.